src/apps/lwaftr/nh_fwd.lua

```diff
@@ -114,6 +120,10 @@ local function send_ipv6_cache_trigger (r, pkt, mac)
    transmit(r, ipv6_cache_trigger(pkt, mac))
 end
 
+local function random_port ()
+   return math.random(65535)
```

Comment:
The issue says "random source UDP/TCP port", for that I understand a random number in the 0-65535 range. Also if I restrict a port range maybe I'm penalizing certain softwires in the binding-table, so I prefer not to make any assumption.